LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2017 Zulip
```

Comment:
Is 2017 right here?

Reply:
I think it should have been 2012-2017.

Moreover, is Zulip right here? Should it have been "Zulip developers and contributors" (as an alternative to putting the name of the orgs/co's)

Reply:
I agree - the main repo reads "Copyright 2011-2017 Dropbox, Inc., Kandra Labs, Inc., and contributors", so maybe we should use this?